tutor/src/models/course/student.js

```diff
@@ -12,6 +12,8 @@ export default
 class CourseStudent extends BaseModel {
   @identifier id;
 
+  static TEACHER_AS_STUDENT_ID = -9;
```

Comment:
Hmm I *think* a negative number will never appear and so is OK...
Would be better if this was a const but I googled and seem that static consts don't exist...

Reply:
hah, good idea.  we actually have a @readonly decorator I can use. that'd be good